SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Notice
+
+The Security Notice for the Python Discord projects can be found [on our website](https://pydis.com/security.md).
```

Comment:
What do you think about dropping the article?

This would read a little better to me: 'The Security Notice for Python Discord projects ...'

Reply:
That does read better, thanks!